### PR TITLE
Backend created

### PR DESCRIPTION
**Pull request by zeeshan4002911**
Backend created  with endpoints for register and login with authentication middleware

### CODE REVIEW

File: constants/dataProduct.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    {
+        id : 1,
+        name: "Shirts",
+        description: "Any color, print type and material",
+        price: 20
+    },
+    {
+        id : 2,
+        name: "T-Shirts",
+        description: "Any color, print type and material",
```
Comment (NOORMOHAMMED789) [inline constants/dataProduct.js:11]: add the proper comments

File: constants/dataProduct.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+const products = [
+    {
+        id : 1,
+        name: "Shirts",
+        description: "Any color, print type and material",
+        price: 20
+    },
+    {
+        id : 2,
+        name: "T-Shirts",
```
Comment (NOORMOHAMMED789) [inline constants/dataProduct.js:10]: dummy comment

Comment (zeeshan4002911) [reply constants/dataProduct.js:10]: dummy reply

**Review by NOORMOHAMMED789 — APPROVED** [review]
done

**Pull request merged**